src/Mafin.Web.Api.Rest/MafinHttpClient.cs

```diff
@@ -280,4 +280,15 @@ public MafinHttpClient(HttpMessageHandler handler, bool disposeHandler)
 #pragma warning restore CA1054
 
     private static CancellationToken ResolveToken(CancellationToken? token) => token ?? CancellationToken.None;
+
+    // Added for interoperability between targets
+    private async Task<HttpResponseMessage> GetCorrectPatchImplementationResponse(Uri? requestUri, HttpContent? content, CancellationToken cancellationToken)
+    {
+#if NETSTANDARD2_1_OR_GREATER
+        var response = await PatchAsync(requestUri, content, cancellationToken).ConfigureAwait(false);
+#else
+        var response = await SendAsync(new HttpRequestMessage(new HttpMethod("PATCH"), requestUri) { Content = content }, cancellationToken).ConfigureAwait(false);
```

Comment:
`HttpMethod.Patch` can be used instead of `new HttpMethod("PATCH")` if I remember correctly.

Reply:
You remember correctly, but the property exists only since netstandard2.1, much like PatchAsync().
So, in this case, it's not applicable unfortunately.

Reply:
Ah. all right.